docs/tendermint-core/running-in-production.md

```diff
@@ -4,6 +4,8 @@ order: 4
 
 # Running in production
 
+If you are running Tendermint in production make sure to use a tag instead of a branch.
```

Comment:
```suggestion
If you are building Tendermint from source for use in production, make sure to check out an appropriate Git tag instead of a branch.
```